tailwind.config.js

```diff
@@ -3,7 +3,16 @@ module.exports = {
   // https://tailwindcss.com/docs/just-in-time-mode
   // mode: "jit",
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        'gray': '#575151',
+        'dark-gray': '#2B2B2B',
+        'dark': '#181719',
+      },
+      fontFamily: {
+        'poppins': ['poppins', 'sans-serif'],
```

Comment:
I think you need to import this from google font correct? How are you doing this on this project?

Reply:
As I am not using it already, I did not imported the font from Google yet. As for the Games API project, the import can be made in the component that it is being utilized.

Reply:
Okay